plugins/modules/k8s_scale.py

```diff
@@ -163,17 +187,9 @@
 }
 
 
-def execute_module(
-    module, k8s_ansible_mixin,
-):
-    k8s_ansible_mixin.set_resource_definitions(module)
-
-    definition = k8s_ansible_mixin.resource_definitions[0]
-
-    name = definition["metadata"]["name"]
-    namespace = definition["metadata"].get("namespace")
-    api_version = definition["apiVersion"]
-    kind = definition["kind"]
+def execute_module(client, module):
+    namespace = module.params.get("namespace")
+    name = module.params.get("name")
```

Comment:
This needs to call `create_definitions()` from `resource.py` first, and then get `namespace`, `name`, `api_version` and `kind` from the returned definition.